types/react-hooks-helper/index.d.ts

```diff
@@ -0,0 +1,33 @@
+// Type definitions for react-hooks-helper 1.6
```

Comment:
I noticed that this package also distributes a umd module. You might want to add a namespace for that. `export as namespace myLibl;`.

Check [this](https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html) template for details.

Reply:
done!